src/utils/useProjectId.ts

```diff
@@ -24,7 +23,7 @@ export const useProjectId = (): [
   projectIdUpdated: boolean,
   clearProjectIdUpdated: () => void
 ] => {
-  const [projectId, setProjectId] = useSharedState<string | null>(
+  const [projectId, setProjectId] = useAddonState<string | null>(
```

Comment:
I guess this doesn't need the workaround because we never really set the projectId to null